src/data_processing/README.md

```diff
@@ -0,0 +1,13 @@
+# Data for visualization
+
+The data to be visualized using `force-graph` can be found in `graph_data_input_file.old.json`. The JSON data contains two types of objects: nodes and links.

```

Comment:
Please explain what`force-graph`is?

Reply:
Ok. I'll briefly explain what `force-graph` is and add a link to the Github repo of the project.